pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -10,17 +10,20 @@ let
     src = fetchFromGitHub {
       owner = "supermerill";
       repo = "SuperSlicer";
-      sha256 = "sha256-Gg+LT1YKyUGNJE9XvWE1LSlIQ6Vq5GfVBTUw/A7Qx7E=";
+      sha256 = "sha256-W+0lbwWoaqaKqUWyFdETi6F/5dMfad+aUhNohG7O+Mg=";
       rev = version;
       fetchSubmodules = true;
     };
 
     # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
     postInstall = null;
```

Comment:
```suggestion
    # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
    dontInstall = true;
```

Reply:
No, I thought you where overwriting installPhase but we should not need to overwrite postInstall with null.

Reply:
I need to disable it somehow — prusa-slicer derivation I override contain postInstall incompatible with mine